### PR TITLE
Add local web mode

### PR DESCRIPTION
**Pull request by ArgentFalcon**
Add ability for webserver to read DAGs locally

I'll make an upstream PR for this shortly, but I wanted to get unblocked before the upstream PR process.

### CODE REVIEW

**Comment by ArgentFalcon** [comment]
👀 @youngyjd @milton0825 @feng-tao 
cc @lyft/dp-tools-viz

**Comment by youngyjd** [comment]
setting this to False would not work? https://github.com/lyft/airflowinfra/blob/master/ops/config/states/airflow.cfg#L183

**Comment by ArgentFalcon** [comment]
Nope. I had to trace through the code a bit to figure out what was happening, but the home page for the views.py never references the loaded DAGBag, only the model found in the database. It still can use the dag definition files on disc to actually read the graph and code

the serialized dag stuff is about not even having the code for the DAGS on disc, the webserver would just read it directly from the DB

File: airflow/config_templates/default_airflow.cfg [hunk 1]
```diff
@@ -434,6 +434,10 @@ force_log_out_after = 0
 # The UI cookie lifetime in days
 session_lifetime_days = 30
 
+# Have the webserver save the dagbag to the DB every refresh. Set to true if you're
```
Comment (astahlman) [inline airflow/config_templates/default_airflow.cfg:437]: Refresh of what? A web page, i.e, on any GET request? Or when the gunicorn master restarts?

File: airflow/config_templates/default_airflow.cfg [hunk 2]
```diff
@@ -434,6 +434,10 @@ force_log_out_after = 0
...
+# running without a scheduler and want DAGS to show up in the webserver
+locally_sync_dagbag = False
```
Comment (astahlman) [inline airflow/config_templates/default_airflow.cfg:439]: The "locally" part tripped me up for a second - maybe just "sync_dagbag_to_db"?

Comment (ArgentFalcon) [reply airflow/config_templates/default_airflow.cfg:439]: names are hard.

**Comment by ArgentFalcon** [comment]
Well, shit:
https://github.com/apache/airflow/issues/8838

**Comment by ArgentFalcon** [comment]
I'm going to get this installed on tars, and test it there, rather than trying to fix this right now. The equivalent change passes tests upstream.

**Pull request merged**